dialect.go

```diff
@@ -310,13 +310,22 @@ func (m ClickHouseDialect) dbVersionQuery(db *sql.DB) (*sql.Rows, error) {
 }
 
 func (m ClickHouseDialect) insertVersionSQL() string {
-	return fmt.Sprintf("INSERT INTO %s (version_id, is_applied) VALUES ($1, $2)", TableName())
+	// TODO(mf): how did this work before?
```

Comment:
I think this was a change in behaviour between `v1` and `v2` need to confirm whether the `v1` driver was special casing bool

Reply:
https://github.com/ClickHouse/clickhouse-go/issues/621